mcs/class/System/net_4_x_System.dll.sources

```diff
@@ -143,6 +143,24 @@ System.IO/InotifyWatcher.cs
 System.IO/KeventWatcher.cs
 System.IO/SearchPattern.cs
 System.IO/WindowsWatcher.cs
+System.IO/CoreFXFileSystemWatcherProxy.cs
+../../../external/corefx/src/System.IO.FileSystem.Watcher/src/System/IO/PatternMatcher.cs
+../../../external/corefx/src/Common/src/Interop/Windows/Interop.Errors.cs
```

Comment:
This should go to windows .sources only and similarly for other Unix/OSX/Windows .cs files